docs/sources/auth/google.md

```diff
@@ -19,7 +19,7 @@ To enable the Google OAuth2 you must register your application with Google. Goog
 First, you need to create a Google OAuth Client:
 
 1. Go to https://console.developers.google.com/apis/credentials
```

Comment:
```suggestion
1. Go to https://console.developers.google.com/apis/credentials.
```

Reply:
Thank you for the suggestion, please merge